// Copyright 2015 OpenLAN authors.

syntax = "proto3";

package ope;

service Ope {
  rpc SayHi (HiRequest) returns (HiReply) {}
  rpc GetCpe(CpeRequest) returns (stream CpeReply) {}

}

message HiRequest {
  string name = 1;
}

message HiReply {
  string message = 1;
}

message CpeRequest {
  string name = 1;
}

message CpeReply {
  string name = 1;
  string up_time = 2;
  sint32 socket = 3;
  uint32 rx_pkt = 4;
  uint32 rx_drop = 5;
  uint32 tx_pkt = 6;
  uint32 tx_drop = 7;
}

